### ISSUE TITLE
Can i run Monopogen for non-tumor sample?

### ISSUE THREAD

**Issue by ZehangJiang** [KChen-lab/Monopogen#67]
Hi, thank u for developing this tools, i'm wondering how we can run Monopogen for non-tumor samples? If so, which pipeline should i choose?

**Comment by jinzhuangdou** [comment]
Yes. You can run Monopogen on normal samples. You can run both germline and somatic module based on your interests.
